nextest-filtering/src/parsing.rs

```diff
@@ -333,11 +332,6 @@ fn parse_matcher_text(input: Span) -> IResult<Option<String>> {
     )(input.clone())
     {
         Ok((i, res)) => (i, res.flatten()),
-        Err(nom::Err::Incomplete(_)) => {
-            let i = input.slice(input.fragment().len()..);
-            // No need for error reporting, missing closing ')' will be detected after
```

Comment:
@sunshowers it seems like this is intentional but no tests are affected.  Any advice on how to hit this case?

Reply:
Not entirely sure! I wasn't the primary author of this code. Will have to investigate to figure this out (may take a few days, I'm really busy sorry!)

Reply:
Fully understand!  Thanks for any help!

Reply:
Yeah, I just spent some time looking at this and you're right.